sklearn/ensemble/_forest.py

```diff
@@ -614,19 +615,18 @@ def predict(self, X):
         if self.n_outputs_ == 1:
             return self.classes_.take(np.argmax(proba, axis=1), axis=0)
 
-        else:
-            n_samples = proba[0].shape[0]
-            # all dtypes should be the same, so just take the first
-            class_type = self.classes_[0].dtype
-            predictions = np.empty((n_samples, self.n_outputs_),
-                                   dtype=class_type)
+        n_samples = proba[0].shape[0]
+        # all dtypes should be the same, so just take the first
+        class_type = self.classes_[0].dtype
+        predictions = np.empty((n_samples, self.n_outputs_),
+                               dtype=class_type)
 
-            for k in range(self.n_outputs_):
-                predictions[:, k] = self.classes_[k].take(np.argmax(proba[k],
-                                                                    axis=1),
-                                                          axis=0)
+        for k in range(self.n_outputs_):
+            predictions[:, k] = self.classes_[k].take(np.argmax(proba[k],
```

Comment:
If you are touching this line, the following will be more readable
```python
predictions[:, k] = self.classes_[k].take(
    np.argmax(proba[k], axis=1), axis=0
)
```

Reply:
thanks!